### PR TITLE
Fix upgrade docs and added stop + uninstall

### PR DESCRIPTION
**Pull request by eldadfux**
Added intructions for stopping and uninstalling Appwrite. Clarified the location from where to run upgrade and migration of data.

### CODE REVIEW

File: app/views/docs/installation.phtml [hunk 1]
```diff
@@ -61,14 +61,31 @@
...
 <h2><a href="/docs/installation#learnMore" id="learnMore">Learn More</a></h2>
 
 <ul>
     <li class="margin-bottom-tiny"><a href="/docs/production" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Running Appwrite in production</a></li>
     <li class="margin-bottom-tiny"><a href="/docs/environment-variables" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Appwrite server environment variables</a></li>
     <li class="margin-bottom-tiny"><a href="/docs/email-delivery" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Appwrite setting a custom SMTP server</a></li>
 </ul>
+
+
+docker-compose rm
```
Comment (Meldiron) [inline app/views/docs/installation.phtml:91]:
```suggestion
```

Left-overs, probably

File: app/views/docs/installation.phtml [hunk 2]
```diff
@@ -61,14 +61,31 @@
     <pre class="line-numbers"><code class="prism language-bash" data-prism>docker-compose up -d --remove-orphans</code></pre>
 </div>
 
-<hr class="margin-bottom" />
```
Comment (Meldiron) [inline app/views/docs/installation.phtml:64]: We removed HR, but never put it back. Should we put it back after uninstalling section?

**Review by Meldiron — CHANGES_REQUESTED** [review]

File: app/views/docs/installation.phtml (hunk 2 above)
Comment (eldadfux) [reply app/views/docs/installation.phtml:64]: I think we can skip it

**Review by Meldiron — APPROVED** [review]
LGTM 💪

**Pull request merged**